components/function-controller/internal/git/go2git_test.go

```diff
@@ -65,10 +65,10 @@ func TestNewGit2Go_LastCommit(t *testing.T) {
 		t.Run(testcase.name, func(t *testing.T) {
 			repoPath := prepareRepo(t)
 			defer deleteTmpRepo(t, repoPath)
-			cloner := &git2goClonerMock{repoPath: repoPath}
+			fetcher := &git2goFetcherMock{repoPath: repoPath}
 
-			opts := Options{Reference: testcase.refName}
-			client := Git2GoClient{cloner}
+			opts := Options{Reference: testcase.refName, URL: repoPath}
```

Comment:
URL as `repoPath`?

Reply:
Well, the struct member is called `URL` and the value we are passing is a path. Internally git2go accepts paths, http(s) and ssh url values.